python/verticox/likelihood.py

```diff
@@ -102,14 +124,27 @@ def derivative_1(z, params: Parameters, sample_idx: int):
     return first_part + second_part
 
 
+@numba.njit()
+def _filter_relevant_event_times(params, u_event_time):
+    result = []
+
+    for t in params.Rt.keys():
+        if t <= u_event_time:
+            result.append(t)
```

Comment:
Is `numba` able to compile appending to a python list? I thought it wouldn't handle most python objects properly

Reply:
Lists, including its methods, are supported in numba: https://numba.pydata.org/numba-doc/dev/reference/pysupported.html#list